src/rules/at-rule-name-newline-after/__tests__/index.js

```diff
@@ -0,0 +1,460 @@
+    code: "@import\n\"x.css\" screen and (orientation:landscape);",
+  }, {
+    code: "@import\r\n\"x.css\" screen and (orientation:landscape);",
+  }, {
+    code: "@import\nurl(\"x.css\");",
+  }, {
+    code: "@importn\r\nurl(\"x.css\");",
+  }, {
+    code: "@import\nurl(\"x.css\") screen and (orientation:landscape);",
+  }, {
+    code: "@import\r\nurl(\"x.css\") screen and (orientation:landscape);",
+  }, {
+    code: "@namespace\nurl(XML-namespace-URL);",
+  }, {
+    code: "@namespace\r\nurl(XML-namespace-URL);",
+  }, {
+    code: "@media\n(min-width: 700px) and (orientation: landscape) { }",
+  }, {
+    code: "@media\r\n(min-width: 700px) and (orientation: landscape) { }",
+  }, {
+    code: "@media\n(min-width: 700px)\nand (orientation: landscape) { }",
+  }, {
+    code: "@media\r\n(min-width: 700px)\r\nand (orientation: landscape) { }",
+  }, {
+    code: "@media\n(min-width: 700px) and\n(orientation: landscape) { }",
+  }, {
+    code: "@media\r\n(min-width: 700px) and\r\n(orientation: landscape) { }",
+  }, {
+    code: "@media\n(min-width: 700px) and (orientation: landscape)  { }",
+  }, {
+    code: "@media\r\n(min-width: 700px) and (orientation: landscape)  { }",
+  }, {
+    code: "@media\n(min-width: 700px) and (orientation: landscape)\n{ }",
+  }, {
+    code: "@media\r\n(min-width: 700px) and (orientation: landscape)\n{ }",
+  }, {
+    code: "@media\n(min-width: 700px) and (orientation: landscape)\r\n{ }",
+  }, {
+    code: "@media\r\n(min-width: 700px) and (orientation: landscape)\r\n{ }",
+  }, {
+    code: "@media\n(min-width: 700px)  and (orientation: landscape) { }",
+  }, {
+    code: "@media\r\n(min-width: 700px)  and (orientation: landscape) { }",
+  }, {
+    code: "@media\n(min-width: 700px)\nand (orientation: landscape) { }",
+  }, {
+    code: "@media\r\n(min-width: 700px)\nand (orientation: landscape) { }",
+  }, {
+    code: "@media\n(min-width: 700px)\r\nand (orientation: landscape) { }",
+  }, {
+    code: "@media\r\n(min-width: 700px)\r\nand (orientation: landscape) { }",
+  }, {
+    code: "@supports\n(animation-name: test) { }",
+  }, {
+    code: "@supports\r\n(animation-name: test) { }",
+  }, {
+    code: "@keyframes\nidentifier { }",
+  }, {
+    code: "@keyframes\r\nidentifier { }",
+  }, {
+    code: "@keyframes\r\nidentifier\n{ }",
+  }, {
+    code: "@keyframes\r\nidentifier\r\n{ }",
+  }, {
+    code: "@viewport { }",
+  }, {
+    code: "@viewport{ }",
+  }, {
+    code: "@viewport\n{ }",
+  }, {
+    code: "@viewport\r\n{ }",
+  }, {
+    code: "@viewport\n\n{ }",
+  }, {
+    code: "@viewport\r\n\r\n{ }",
+  }, {
+    code: "@counter-style\nwinners-list { }",
+  }, {
+    code: "@counter-style\r\nwinners-list { }",
+  }, {
+    code: "@font-face { };",
+  }, {
+    code: "@unknown\n\"ident\";",
+  }, {
+    code: "@unknown\r\n\"ident\";",
+  }, {
+    code: "@unknown\nident { };",
+  }, {
+    code: "@unknown\r\nident { };",
+  }, {
+    code: "a { color: pink; @crazy-custom-at-rule; }",
```

Comment:
Why is this one ok? What is the exception here? It should be documented, I think.